src/FantasyFun.API/Controllers/PlayersController.cs

```diff
@@ -0,0 +1,78 @@
+﻿using FantasyFun.API.Repositories;
+using FantasyFun.API.Settings;
+using FantasyFun.API.ViewModel;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using System.Linq;
+
+namespace FantasyFun.API.Controllers
+{
+
+
+    [Route("api/[controller]")]
+    [ApiController]
+    public class PlayersController : ControllerBase
+    {
+        private readonly DateTime _defaultGameTime;
+
+        private readonly FootballDbContext _dbContext;
+
+        public PlayersController(FootballDbContext dbContext, GameSettings gameSettings)
+        {
+            _dbContext = dbContext;
+            _defaultGameTime = gameSettings.DefaultGameTime;
+        }
+
+        [HttpGet]
+        [Route("overall/{overall}")]
+        public async Task<ActionResult<List<string>>> GetAnyPlayers(long overall)
+        {
+            //var db = new FootballDbContext();
+
+            var anyPlayers = await _dbContext.PlayerAttributes
+                .Where(l => l.Date <= _defaultGameTime)
+                .OrderByDescending(l => l.Date)
+                .GroupBy(l => l.Player.Name)//.Max(l => l.Select(k => k.Date))
+                .Select(l => new PlayerType(l.FirstOrDefault().Player.Name, l.FirstOrDefault().OverallRating, l.FirstOrDefault().Date))
+                .Take(500)
```

Comment:
You can make 500 as private variable inside this class. You can name it 'InternalPlayersByOverallMaxNumber'.